### PR TITLE
Build PRs

### PR DESCRIPTION
**Pull request by matiasgarciaisaia**
- Fixed missing plugins during build
- Build PRs but only deploy branches

### CODE REVIEW

**Comment by matiasgarciaisaia** [comment]
@bcardiff you won't probably recall, but was there any good reason we weren't building PRs?

File: .travis.yml [hunk 1]
```diff
@@ -18,3 +16,6 @@ deploy:
   local_dir: _book
   upload-dir: docs
   skip_cleanup: true
+  on:
+    branch: master
+    condition: "$TRAVIS_PULL_REQUEST = true"
```
Comment (RX14) [inline .travis.yml]: why? I'm confused.

**Comment by RX14** [comment]
Isn't the correct fix to add `gitbook-plugin-edit-link` to our `package.json` similar to the `ga` plugin?

**Comment by straight-shoota** [comment]
I've just created #168 for adding `gitbook install`. I think this should better be in a separate PR because it is not enabling builds for pull requests.

**Comment by straight-shoota** [comment]
@RX14 The recommended installation method for the [`edit-link` plugin](https://plugins.gitbook.com/plugin/edit-link) is to run `gitbook install`. For [`ga`](https://plugins.gitbook.com/plugin/ga) it's using `npm install gitbook-plugin-ga` which is equivalent to adding it to `package.json`. I don't know the difference between both methods.

File: .travis.yml (hunk 1 above)
Comment (matiasgarciaisaia) [reply .travis.yml]: Well spotted. I've mixed the previous build condition with the `||` operator - it should deploy when PR is `false` 👍

**Comment by RX14** [comment]
If that's the case then we should remove the package.json. Personally I prefer to use 1 package manager not 2.

**Review by RX14 — APPROVED** [review]

**Pull request merged**